src/Nethermind/Nethermind.Merge.Plugin/IMergeConfig.cs

```diff
@@ -55,5 +56,8 @@ public interface IMergeConfig : IConfig
 
         [ConfigItem(Description = "Requests the GC to release process memory back to OS. Accept values `-1` which disables it, `0` which releases every time, and any positive integer which does it after that many EngineApi calls.", DefaultValue = "75")]
         public int CollectionsPerDecommit { get; set; }
+
+        [ConfigItem(Description = "Maximum time in seconds for NewPayload request to be executed.")]
```

Comment:
Default value? hide from docs?

Reply:
Default was forced in code but yeah actually it is good to set it from config as well.
HideFromDocs - very good point, added, thanks!